src/sidebar/components/markdown-editor.js

```diff
@@ -293,7 +293,7 @@ export default function MarkdownEditor({
           dir="auto"
           ref={input}
           onClick={e => e.stopPropagation()}
-          onKeydown={handleKeyDown}
+          onKeyDown={handleKeyDown}
```

Comment:
I wasn't sure whether the case mattered here. After a little research I discovered that. known event names are [case-insensitive in Preact](https://github.com/preactjs/preact/blob/33a8c15d78fb9db2f3f090b2781b466b6b86042d/src/diff/props.js#L98). From a quick skim through the React repo and docs, it seems like this casing (`onKeyDown` rather than `onKeydown` as it was before) is more idiomatic.

Reply:
Thanks for looking that up, my hunch was to keep it consistent